src/js/tech/tech.js

```diff
@@ -218,12 +218,13 @@ class Tech extends Component {
       return;
     }
 
+    let player = this.player();
     let textTracksChanges = function() {
       let updateDisplay = Fn.bind(this, function() {
-        this.trigger('texttrackchange');
+        player.trigger('texttrackchange');
```

Comment:
Are you sure this is needed? There's [no player anymore](https://github.com/videojs/video.js/pull/2215/files#diff-9242a070ce50900e482a6a680f51e25cR28) in techs, so we need to find another option. The player does [listen for texttrackchange events](https://github.com/videojs/video.js/blob/a6a4a0fd242ef08f97d7abec4561bb15f69c0616/src/js/player.js#L485) on the tech. It this is still actually triggering the event on the tech, because the component init is setting the tech as player.